.travis.yml

```diff
@@ -11,3 +11,4 @@ script:
   # TODO(rhcarvalho): check syntax of other important entrypoint playbooks
   - ansible-playbook --syntax-check playbooks/byo/config.yml
   - cd utils && make ci
+  - make test-integration
```

Comment:
We have already some integration with vagrant-openshift to run these tests in our Jenkins, but before we transition it to point to openshift/openshift-ansible, Travis can demo a test run.